### ISSUE TITLE
How to get a probability for the result of t5_tokenizer.decode(output,...)?

### ISSUE THREAD

**Issue by LianaN** [huggingface/transformers#10800]
Hello,

I am using `t5-base` to map phrases into categories, for example: "I want to eat" -> "hunger".
Is there any way to get the probability for `result` values?

For example, if the input is "He is hungry", the model returns 5 labels. These results seem to be ordered by some relevance rank, so that the most relevant label is always first in `outputs`. So, my question is how can I retrieve these probabilities?

My final goal is to set a threshold on the probability, so that `outputs` would only include results that pass this threshold, or it can be empty if nothing relevant found. 

```
t5_tokenizer = T5Tokenizer.from_pretrained('t5-base')
t5_model = T5ForConditionalGeneration.from_pretrained('t5-base')
...
model.model.eval()
outputs = model.model.generate(
    input_ids=test_input_ids,attention_mask=test_attention_mask,
    max_length=64,
    early_stopping=True,
    num_beams=10,
    num_return_sequences=5,
    no_repeat_ngram_size=2
)

for output in outputs:
    result = t5_tokenizer.decode(output, skip_special_tokens=True, clean_up_tokenization_spaces=True)
    print(result)
```

Thanks.

**Comment by LysandreJik** [comment]
Hello, thanks for opening an issue! We try to keep the github issues for bugs/feature requests.
Could you ask your question on the [forum](https://discusss.huggingface.co) instead?

Thanks!
